models/atac_intact_hic/feature_table.tsv

```diff
@@ -0,0 +1,9 @@
+feature	input_col	second_input	aggregate_function	fill_value	nice_name
+numTSSEnhGene	numTSSEnhGene		max	0	# TSSs between E and P
+distanceToTSS	distance		min		Distance to TSS
+normalizedATAC_prom	normalized_atac_prom		mean	0	ATAC signal at P
```

Comment:
Perhaps we should change this as well since we know ATAC at promoter isn't a good feature, at least for the encode datasets?